src/main/java/de/eddyson/tapestry/react/services/nashorn/NashornReactRenderEngine.java

```diff
@@ -0,0 +1,16 @@
+package de.eddyson.tapestry.react.services.nashorn;
+
+import org.apache.tapestry5.json.JSONObject;
+
+public interface NashornReactRenderEngine {
```

Comment:
Can we use a more generic name for the interface? I can imagine a Rhino-based implementation as well as one that uses node.